plugins/GSdx/GSWndWGL.cpp

```diff
@@ -323,16 +331,45 @@ void* GSWndWGL::GetProcAddress(const char* name, bool opt)
 
 //TODO: check extensions supported or not
 //FIXME : extension allocation
-void GSWndWGL::SetVSync(int vsync)
+void GSWndWGL::SetSwapInterval(int vsync)
 {
 	// m_swapinterval uses an integer as parameter
 	// 0 -> disable vsync
 	// n -> wait n frame
 	if (m_swapinterval) m_swapinterval(vsync);
 }
 
+bool GSWndWGL::CompositorEnabled()
+{
+	BOOL dwm;
+	if (DwmIsCompositionEnabled(&dwm) != S_OK)
+		return false;
+
+	// Note: DWM is always true on Win8 and later
+	if (!dwm)
+		return false;
+
+	// Easy case are done. It remains driver black magic
+
+	// Recent driver (+380.xx ??) seem to always get v-sync enforced someway.
+	// XXX it would be nice to understand what happen to fix it properly
+	if (GLLoader::nvidia_buggy_driver)
+		return true;
```

Comment:
It means the mindblowing mystery that I tried to figure out in #1437
First I thought it was about exclusive mode not triggering. Than it seemed to be about v-sync always being enforced. 
Now.. Hell I'd knew. 

I totally trust you'll be able to determine whenever it should be the case we applied DWMFlush (and set swap_interval to 0) and when not.